src/calculator.js

```diff
@@ -1,47 +1,28 @@
-exports._check = () => {
-  // DRY up the codebase with this function
-  // First, move the duplicate error checking code here
-  // Then, invoke this function inside each of the others
-  // HINT: you can invoke this function with exports._check()
+exports._check = (...items) => {
+  items.forEach(item => {
```

Comment:
Expected parentheses around arrow function argument having a body with curly braces.

###### See http://eslint.org/docs/rules/arrow-parens for details about this rule